src/tokens/eth/0x515bA0a2E286AF10115284F151cF398688A69170.json

```
@@ -0,0 +1,26 @@
+{
+  "symbol": "TENX",
+  "name": "TenX",
+  "type": "ERC20",
+  "address": "0x515bA0a2E286AF10115284F151cF398688A69170",
+  "ens_address": "",
+  "decimals": "18",
+  "website": "https://tenx.tech/en/",
+  "logo": { "src": "https://tinypng.com/web/output/12dv3ww04dqc2vmwg8jnq85bhhdbqzxt/TENXtoken256_2.png", "width": "256", "height": "256", "ipfs_hash": "" },
+  "support": { "email": "support@trusttoken.com", "url": "https://support.tenx.tech/hc/en-us" },
+  "social": {
+    "blog": "https://blog.tenx.tech/",
+    "chat": "https://chat.tenx.tech/",
+    "facebook": "https://www.facebook.com/tenxwallet/",
+    "forum": "",
+    "github": "https://github.com/tenx-tech/",
```

```suggestion
    "github": "https://github.com/tenx-tech/tenx-token",
```
Correct github repo for the TenX token.